tests/netflix/test_cassandra.py

```diff
@@ -78,70 +81,70 @@ def test_capacity_small_fast():
         assert small_result.cluster_params["cassandra.heap.table.percent"] == 0.11
 
 
-def test_ebs_high_reads():
```

Comment:
if you did it with the param then you can just pass required_local_disks false to these tests and keep the context where EBS _would_ be useful.

Reply:
That's a good point. However, I do want to preserve the old tests as this is a temporary change. And hence commenting out the tests could be a better choice?

Reply:
If I set `"require_local_disks": False` and `"require_attached_disks": True, ` in the test then it will test for ebs case. So we are achieving the result of old tests just by passing extra params.